scripts/dependencies/lima.ts

```diff
@@ -50,6 +53,31 @@ export class LimaAndQemu implements Dependency, GithubDependency {
         }
       });
     });
+
+    // Download limactl built with the specified macOS codename
+    if (limactlCodename != '') {
+      const limactlUrl = `${ baseUrl }/v${ context.versions.limaAndQemu }/limactl.${ limactlCodename }.tar.gz`;
+      const limactlExpectedChecksum = (await getResource(`${ limactlUrl }.sha512sum`)).split(/\s+/)[0];
+      const limactlDir = path.join(context.resourcesDir, context.platform, 'lima', 'bin');
+      const limactlTarPath = path.join(context.resourcesDir, context.platform, `limactl-v${ context.versions.limaAndQemu }.tgz`);
+
+      await download(limactlUrl, limactlTarPath, {
+        expectedChecksum: limactlExpectedChecksum, checksumAlgorithm: 'sha512', access: fs.constants.W_OK,
+      });
+
+      const limactlChild = childProcess.spawn('/usr/bin/tar', ['-xf', limactlTarPath, `--transform='s/limactl/limactl.${ limactlCodename }/'`],
```

Comment:
macOS is bundling bsdtar, but `--transform` is a gnutar option. Use `-s` instead!

Reply:
Good call out. Will address tomorrow.